packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { actions } from 'data'
+import { bindActionCreators, compose, Dispatch } from 'redux'
+import { connect } from 'react-redux'
+import React, { PureComponent } from 'react'
+import Template from './template'
+
+export type OwnProps = {
+  handleClose: () => void
+}
+
+export type LinkDispatchPropsType = {
+  identityVerificationActions: typeof actions.components.identityVerification
+  simpleBuyActions: typeof actions.components.simpleBuy
+}
+
+type LinkStatePropsType = {}
+type Props = OwnProps & LinkDispatchPropsType & LinkStatePropsType
+type State = {}
+
+class KycRequired extends PureComponent<Props, State> {
+  state = {}
+
+  render () {
+    return <Template {...this.props} />
+  }
+}
+
+const mapStateToProps = (): LinkStatePropsType => ({})
+
+const mapDispatchToProps = (dispatch: Dispatch): LinkDispatchPropsType => ({
+  simpleBuyActions: bindActionCreators(actions.components.simpleBuy, dispatch),
+  identityVerificationActions: bindActionCreators(
+    actions.components.identityVerification,
+    dispatch
+  )
+})
+
+const enhance = compose(connect(mapStateToProps, mapDispatchToProps))
+
+export default enhance(KycRequired)
```

Comment:
I think maybe you're using an older snippet? Can you update to the latest here? https://github.com/blockchain/blockchain-wallet-v4-frontend/wiki/Helpful-.tsx-Snippets#typescript-react-pure-component-for-remote-types